x-pack/plugins/apm/server/lib/search_strategies/correlations/queries/query_field_value_pairs.ts

```diff
@@ -71,7 +71,7 @@ export const fetchTransactionDurationFieldValuePairs = async (
         .attribute_terms as estypes.AggregationsMultiBucketAggregate<{
         key: string;
       }>)?.buckets;
-      if (buckets.length >= 1) {
+      if (buckets.length >= 0) {
```

Comment:
Was this meant to be `> 0`?

Reply:
Looking at it again it might be more save to use `buckets?.length` here

Reply:
Thanks for pointing this out, I got confused with the old code, it was alright actually. Just added the `?` in 99b1976.